R/github-actions.R

```diff
@@ -152,15 +118,97 @@ use_github_action <- function(name,
     ui_todo("Learn more at <{readme}>.")
   }
 
+  badge <- badge %||% is_check_action(url)
+  if (badge) {
+    use_github_actions_badge(save_as)
+  }
+
   invisible(new)
 }
 
-#' @section `use_github_action_check_release()`:
-#' This entry-level, bare-minimum workflow installs the latest release of R (on
-#' a current distribution of Linux) and runs `R CMD check` via the
-#' [rcmdcheck](https://github.com/r-lib/rcmdcheck) package.
+choose_gha_workflow <- function(error_call = caller_env()) {
+  if (!is_interactive()) {
+    cli::cli_abort(
+      "{.arg name} is absent and must be supplied",
+      call = error_call
+    )
+  }
+
+  prompt <- cli::format_inline(
+    "Which action do you want to add? (0 to exit)\n",
+    "(See {.url https://github.com/r-lib/actions/tree/v2/examples} for other options)"
+  )
+  # Any changes here also need to be reflected in documentation
+  workflows <- c(
+    "check-standard" = "Run R CMD check in four environments",
+    "test-coverage" = "Compute test coverage and report to https://about.codecov.io",
+    "lint" = "Run lintr::lint_package() and report problems",
+    "pr-commands" = "Add /document and /style commands for pull requests"
+  )
```

Comment:
It would be good to put some thought into what we should list here. Maybe we should have separate lists depending on whether or not `is_package()` is `TRUE `?

Reply:
pkgdown comes to mind

Reply:
I left it off because I thought we'd want to steer people towards `use_pkgdown_github_pages()`? Either way it's probably worth mentioning that in the docs.